survey_answer_generation/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Survey answer generation",
+    "summary": "Pre-generate answers from another survey",
```

Comment:
It's more than just pre-generate

```suggestion
    "summary": "Link and compare answers from another survey",
```